keyboards/handwired/aek64/keymaps/4sstylz/config.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
```

Comment:
license header?

Reply:
Doesn't seem consistent across this keymap anyway.